.gitignore

```diff
@@ -40,3 +40,4 @@ build/
 spec/env_variables.rb
 screenshots/
 output/
+spec/examples.txt
```

Comment:
@a-ashbeck make sure you add this to `lutgendorf_features/.gitignore` this file is created due to the updates in the spec_helper